apps/arweave/src/ar_block.erl

```diff
@@ -526,7 +605,9 @@ hash_wallet_list(WalletList) ->
 						(ar_serialize:encode_bin(LastTX, 8))/binary,
 						(ar_serialize:encode_int(Denomination, 8))/binary,
 						MiningPermissionBin/binary >>,
-				crypto:hash(sha384, Preimage)
+				Alg = case Height >= ar_fork:height_2_8() of true -> sha256;
```

Comment:
I forget the context on this change. Is this required for the packing depth changes, or we taking the opportunity to make this change since we have to do a hard fork anyways?

And the rationale is to reduce the size of the wallet list?